test/parallel/test-quic-client-connect-err.js

```diff
@@ -72,8 +78,130 @@ const client = createSocket();
 });
 
 // Test invalid alpn argument option
-[-1, 10, 1n, {}, []].forEach((alpn) => {
+[-1, 10, 1n, {}, [], true].forEach((alpn) => {
   assert.throws(() => client.connect({ alpn }), {
     code: 'ERR_INVALID_ARG_TYPE'
   });
 });
+
+[
+  'idleTimeout',
+  'activeConnectionIdLimit',
+  'maxAckDelay',
+  'maxData',
+  'maxPacketSize',
+  'maxStreamDataBidiLocal',
+  'maxStreamDataBidiRemote',
+  'maxStreamDataUni',
+  'maxStreamsBidi',
+  'maxStreamsUni',
+].forEach((prop) => {
+  assert.throws(() => client.connect({ [prop]: -1 }), {
+    code: 'ERR_OUT_OF_RANGE'
+  });
+
+  ['a', 1n, [], {}, false, Number.MAX_SAFE_INTEGER + 1].forEach((val) => {
+    assert.throws(() => client.connect({ [prop]: val }), {
+      code: 'ERR_INVALID_ARG_TYPE'
+    });
+  });
+});
+
+// activeConnectionIdLimit must be between 2 and 8, inclusive
+[1, 9].forEach(( activeConnectionIdLimit ) => {
```

Comment:
Nice to see the linter works now :)

```suggestion
[1, 9].forEach((activeConnectionIdLimit) => {
```

Reply:
The js linter works great. c++ linter not so much